tests/unit_tests/storage/test_extraction.py

```diff
@@ -163,10 +163,10 @@ def test_empty_ensemble_with_name(client):
 
     # Create case with given name
     ert = ErtConfigBuilder().build()
-    ert.select_or_create_new_case(name)
+    ert._enkf_main.storage_manager.add_case(name)
```

Comment:
`ert._enkf_main.`.. The return from `ErtConfigBuilder` is a `LibresFacade` instance (kinda got confused with `ert =` there).

Could this warrant expanding the `LibresFacade` api? To include exposing `add_case`? If it makes sense that it expose `select_or_create..`, then I would think it could make sense to also expose `add_case`..

Would be nice not having to access a private member (and especially not` enkf_main`)

Reply:
I was thinking we could try to abstract away this somewhat, but obviously we need it at some level, let us discuss it tomorrow.